src/xz/chevahbs

```diff
@@ -37,7 +37,10 @@ chevahbs_try() {
 
 
 chevahbs_cp() {
```

Comment:
this needs a comment ...  I have no idea why we need this and what it does

Reply:
It's copying needed files after a successful build. All the commands in it for this library are also `cp` commands.